client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -36,6 +36,9 @@ const transformApiRequest = ( { data: jitms } ) =>
 		CTA: {
 			message: unescapeDecimalEntities( jitm.CTA.message ),
 			link: unescapeDecimalEntities( jitm.CTA.link || '' ),
+			target: unescapeDecimalEntities(
+				jitm.CTA.target || '' === jitm.CTA.target ? jitm.CTA.target : '_blank'
```

Comment:
`''` is falsy, so the ternary isn't necessary in this case:
```suggestion
				jitm.CTA.target || '_blank'
```

Reply:
Thanks @gcsecsey, we don't want to return '_blank' when "target" is an empty string. In some situations especially when we want to load content without a full page reload, having a target forces a refresh.

Reply:
Oh I see, thanks for clarifying!